assets/src/edit-story/components/statusCheck/test/statusCheck.js

```diff
@@ -14,54 +14,71 @@
  * limitations under the License.
  */
 /**
- * Internal dependencies
+ * External dependencies
  */
+import {
+  fireEvent,
+  screen,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import Modal from 'react-modal';
+
 /**
- * External dependencies
+ * Internal dependencies
  */
-import { FlagsProvider } from 'flagged';
 import { renderWithTheme } from '../../../testUtils';
 import ConfigContext from '../../../app/config/context';
 import APIContext from '../../../app/api/context';
 import StatusCheck from '../statusCheck';
-/**
- * External dependencies
- */
 
 function setup(response) {
   const configValue = { api: { statusCheck: '' }, metadata: { publisher: '' } };
 
-  const apiValue = {
-    actions: {
-      getStatusCheck: () => response,
-    },
-  };
+  const getStatusCheck = () => response;
+  const apiValue = { actions: { getStatusCheck } };
 
   const { queryAllByRole, container } = renderWithTheme(
-    <FlagsProvider
-      features={{
-        statusCheck: true,
-      }}
-    >
-      <ConfigContext.Provider value={configValue}>
-        <APIContext.Provider value={apiValue}>
-          <StatusCheck />
-        </APIContext.Provider>
-      </ConfigContext.Provider>
-    </FlagsProvider>
+    <ConfigContext.Provider value={configValue}>
+      <APIContext.Provider value={apiValue}>
+        <StatusCheck />
+      </APIContext.Provider>
+    </ConfigContext.Provider>
   );
   return { queryAllByRole, container };
 }
 
 describe('statusCheck', () => {
-  it('successful', () => {
+  let modalWrapper;
+
+  beforeAll(() => {
+    modalWrapper = document.createElement('aside');
+    document.documentElement.appendChild(modalWrapper);
+    Modal.setAppElement(modalWrapper);
+  });
+
+  afterAll(() => {
+    document.documentElement.removeChild(modalWrapper);
+  });
+
+  it('should do nothing if successfull', () => {
     setup(Promise.resolve({ success: true }));
-    // todo add a test here.
-    expect(true).toBeTrue();
+
+    const dialog = screen.queryByRole('dialog');
+    expect(dialog).toBeNull();
   });
-  it('failure', () => {
+
+  it('should display dismissible dialog if failed', async () => {
     setup(Promise.reject(new Error('api failed')));
-    // todo add a test here.
-    expect(true).toBeTrue();
+
+    await new Promise((r) => setTimeout(r, 1000));
```

Comment:
Curious why the `1000` timeout?

Reply:
Oh, that was just a test of stuff. Will remove again.

Reply:
Removed in 8ad7f9acb.